docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -67,3 +67,53 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
 _Generally available in Grafana Enterprise_
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Transformations
+
+As our work on improving the user experience of transforming data continues, we've also been adding new capabilities to transformations.
+
+### Support for dashboard variables in transformations
+
+<!-- Oscar Kilhed, Victor Marin -->
+
+_Experimental in Grafana Open Source and Enterprise_
+
+Previously, the only transformation that supported [dashboard variables](https://grafana.com/docs/grafana/<GRAFANA VERSION>/dashboards/variables/) was the **Add field from calculation** transformation. We've now extended the support for variables to the **Filter by value**, **Create heatmap**, **Histogram**, **Sort by**, **Limit**, **Filter by name**, and **Join by field** transformations. We've also made it easier to find the correct dashboard variable by displaying available variables in the fields that support them, either in the drop-down or as a suggestion when you type **$** or press Ctrl + Space:
+
+{{< figure src="/media/docs/grafana/transformations/completion.png" caption="Input with dashboard variable suggestions" >}}
+
+### New modes for the Add field from calculation transformation
+
+<!-- Victor Marin -->
+
+_Generally available in Grafana Open Source and Enterprise_
+
+The **Add field from calculation** transformation has two new modes.
+
+**Unary operations** let you apply mathematical operations to a field. The currently supported operations are:
+
+- **Absolute value (abs)** - Returns the absolute value of a given expression. It represents its distance from zero as a positive number.
+- **Natural exponential (exp)** - Returns _e_ raised to the power of a given expression.
+- **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
+- **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
+- **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
+
+{{< figure src="/media/docs/grafana/transformations/unary-operation.png" >}}
+
+**Row index** adds a field that represents the row index of the row.
```

Comment:
I think this needs modification. For this release we only added a new switch `As percentage` to show the existing row index as a percent instead. I can make the modifications here

Reply:
@oscarkilhed let me know what you think of the changes

Reply:
Ah, my mistake, thanks for the correction @mdvictor

Reply:
no worries!